.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
```

Comment:
it should solve the issue https://github.com/AR-js-org/AR.js/issues/616 to be confirmed.